charts/datadog/templates/container-process-agent.yaml

```diff
@@ -22,9 +22,13 @@
     {{- end }}
     - name: DD_LOG_LEVEL
       value: {{ .Values.agents.containers.processAgent.logLevel | default .Values.datadog.logLevel | quote }}
-    {{- if .Values.datadog.systemProbe.enabled }}
+    {{- if or .Values.datadog.systemProbe.enabled .Values.datadog.networkMonitoring.enabled }}
```

Comment:
why not using `{{- if eq (include "should-enable-system-probe" .) "true" }}` here?

Reply:
The process-agent should only be started with the system-probe for the network-check, so turning on just runtime security monitoring should not set  ENABLED=true.